activities.json

```diff
@@ -506,6 +506,18 @@
     "title": "HTTP Client Hints",
     "url": "https://tools.ietf.org/html/draft-ietf-httpbis-client-hints"
   },
+  {
+    "ciuName": null,
+    "description": "The ES Import Attributes and JSON modules proposal adds an inline syntax for module import statements to pass on more information alongside the module specifier, and an initial application for such attributes in supporting JSON modules in a common way across JavaScript environments.",
+    "id": "import-attributes",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1648614",
+    "mozPosition": "worth prototyping",
+    "mozPositionDetail": "This proposal enables importing non-js files into a JS module, specifically in this case JSON. Without this mechanism, it wouldn't be possible to do this in a way that isn't open to midflight changing of the file type, which could then be inteereepreetteed as JS. The proposal is necessary to move forward on HTML/CSS/JSON modules. We consider it important that this is defined inbound rather than out of bound.",
```

Comment:
It's not clear to me what "inbound" and "out of bound" mean.  Might it be worth making this clearer?

Reply:
It should probably say "in band" and "out of band". Even with that change, it may still need more clarity.

Reply:
It seems like this might be too much detail: "This proposal enables the importing of JSON content into a JS module. This mechanism is a prerequisite for HTML/CSS/JSON modules."

The point about security is probably OK to skip, and unless there is something fundamental in the disagreement we have regarding in-band vs. out-of-band, then I would skip that alss.  The security advantage seems like that this is an innate property of the proposal, so I'm not sure why it would be called out in this way.  If you want to say something then I would suggest "Explicit indication of file types as proposed has the advantage of avoiding content type substitution attacks."  or something like that.

Reply:
Ok, I have updated it to reflect this.